src/Repository/NativeAssignmentRepository.php

```diff
@@ -33,14 +31,9 @@
 
 class NativeAssignmentRepository extends AbstractRepository
 {
-    /** @var string */
-    private $kernelEnvironment;
-
-    public function __construct(ManagerRegistry $registry, string $kernelEnvironment)
+    public function __construct(ManagerRegistry $registry)
```

Comment:
Lots of things were removed here. If it's not related to UUID, it would be very kind to describe it on the PR description just to track it if needed or maybe create another PR for it :)

Reply:
It's just technical detail: we don't have to refresh sequences anymore since we don't use them anymore.